packages/common/composable/http/axios.js

```
@@ -0,0 +1,151 @@
+      }
+
+      if (Array.isArray(config)) {
+        config.forEach((item) => {
+          mock.proxy(item)
+        })
+      }
+
+      return mock
+    },
+    disableMock() {
+      mock && mock.restore()
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)